diff --git a/src/osgPlugins/cfg/CMakeLists.txt b/src/osgPlugins/cfg/CMakeLists.txt
index 972675f..4f7062b 100644
--- a/src/osgPlugins/cfg/CMakeLists.txt
+++ b/src/osgPlugins/cfg/CMakeLists.txt
@@ -20,7 +20,7 @@ SET(TARGET_H

 # lex/yacc generated files use register that causes warnings with CLang under OSX so disable this warnings.
 IF(${CMAKE_CXX_COMPILER_ID} STREQUAL "Clang")
-   SET(CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS} -Wno-deprecated-register)
+   SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-deprecated-register")
 ENDIF()

 #
